src/options.json

```diff
@@ -118,6 +118,10 @@
         }
       ]
     },
+    "icss": {
+      "description": "Enables/Disables handling the CSS module interoperable import/export format ((https://github.com/webpack-contrib/css-loader#icss)",
+      "type": "boolean"
+    },
```

Comment:
Put this option to `modules.icss`, because it is related to CSS modules, not for CSS itself

Reply:
that would make the other checks for css modules harder since they all check if `modules` is truthy. e.g.

```js
shouldUseModulesPlugins({ modules }) {
   if (typeof modules === 'object && modules) return options.icss 
   return Boolean(modules)
}
```

Also it's semantically odd, `module: { icss: true }` doesn't enable css modules but any other truthy value would

Reply:
Yep, you are right, let's fix CI